2996_reflection/reflection.md

```diff
@@ -3021,6 +3021,28 @@ Change the grammar for `$operator-or-punctuator$` in paragraph 1 of [lex.operato
 ```
 :::
 
+### [basic.def.odr]{.sref} One-definition rule {-}
+
+Preprend before paragraph 14 of [basic.def.odr]{.sref}:
+
+::: std
+
+::: addu
+
+[14pre]{.pnum} If a class `C` is defined in a translation with a call to `std::meta::define_class`, every definition of that class shall be the result of a call to `std::meta::define_class` such that its respective members are equal in number and have respectively the same types, alignments, [[no_unique_address]] attributes (if any), bit-field widths (if any), and specified names (if any).
+
+:::
+
+:::
+
+Adjust paragraph 14 as follows:
+
+::: std
+
+[14]{.pnum} [Otherwise, for]{.add}[For]{.rm} any definable item D with definitions ...
```

Comment:
```suggestion
[14]{.pnum} [Otherwise, for]{.addu} [For]{.rm} any definable item D with definitions ...
```

Also let's put these together in one block, so that it looks like:

> Insert a new paragraph before basic.def.odr/15:
> 
> ::: std
> ::: addu
> [15pre]{.pnum} This stuff
> :::
> [15]{.pnum} This other stuff
> :::